json-endpoints/class.wpcom-json-api-site-settings-endpoint.php

```diff
@@ -111,6 +112,12 @@ public function get_settings_response() {
 			case 'URL' :
 				$response[$key] = (string) home_url();
 				break;
+			case 'locale_variant':
+				$blog_variant_id = wpcom_l10n_get_blog_variant_lang_id();
```

Comment:
If this runs on a Jetpack site, we need to add a `function_exists` and fallback to `get_option( 'lang_id' )